---
layout: page
title: Current Projects
date: 2017-02-26T00:15:47+01:00
modified: 2017-09-02T12:30:00+01:00
excerpt: "Urminder Singh's current projects"
tags: [Urminder Singh, research, bioinformatics, evolution, machine learning]
---


<section id="table-of-contents" class="toc">
  <header>
    <h3>Overview</h3>
  </header>
<div id="drawer" markdown="1">
*  Auto generated table of contents
{:toc}
</div>
</section><!-- /#table-of-contents -->


### Role of genetic ancestry in COVID-19 susceptibility and severity

As of January 2021, the COVID-19 pandemic has infected over 98 million people and killed over [2 million worldwide](https://coronavirus.jhu.edu/map.html).
People of African and Latin ancestries disproportionately [affected](https://blog.23andme.com/23andme-research/impact-disparities-of-covid-19/) by COVID-19 with respect to prevalence, morbidity, and mortality.

An individual's health is a result of complex interactions among one's [genetic](https://www.sciencemag.org/news/2020/10/found-genes-sway-course-coronavirus), environmental, and [socio-economic](https://blogs.scientificamerican.com/voices/racism-not-genetics-explains-why-black-americans-are-dying-of-covid-19/) factors.
We leveraged RNA-Seq expression data and probed differences in gene-expression among people from European and African Ancestries. Although we could not control environmental and socio-economic factors (because of lack of metadata), we found differential expression of genes implicated in COVID-19 infection.
Further, African American gene expression signatures reveal an increased number or activity of esophageal glandular cells and lung ACE2-positive basal keratinocytes.
Please read our preprint [here](https://assets.researchsquare.com/files/rs-88890/v1_stamped.pdf).

Our ongoing research, in collaboration with the COVID-19 International Research Team ([COV-IRT](https://www.cov-irt.org/)), aims to detect genetic factors that can explain the disparity of COVID-19 infection and mortality rate among different populations. 
We are particularly interested in identifying and examining the roles of human specific [orphan genes](https://elifesciences.org/articles/55136) and non-coding RNAs in COVID-19 infection.

### Origins of novel genes

![](/images/geneslifecycle.jpg)
*Figure showing life cycle of orphan genes. [Ref: Genetic Novelty: How new genes are born](https://elifesciences.org/articles/55136)*

During evolution, genes undergo duplication and divergence and give rise to new genes.
These new genes can potentially code for [new proteins](https://en.wikipedia.org/wiki/Gene_duplication) and may provide the organism with useful biological functions.
Genes resulting through the process of duplication could be traced back to their ancestral genes via [sequence homology](https://en.wikipedia.org/wiki/Sequence_homology).
It has been observed that every in organism a number of genes could not be traced back to an ancestral gene. These gene are termed as [orphan genes](https://en.wikipedia.org/wiki/Orphan_gene). 
Orphan genes can arise [*de novo*](https://en.wikipedia.org/wiki/De_novo_gene_birth) from non-genic regions in the genome. Many such orphan genes are found to be involved in biologically important functions.

Focus of my research is to identify and characterize novel expressed genes in humans. To achieve this goal I am integrating and analyzing terabytes high quality of RNA-Seq datasets from massive projects like [GTEx](https://gtexportal.org/home/), [TCGA](https://portal.gdc.cancer.gov) and [SRA](https://www.ncbi.nlm.nih.gov/sra).




### Examining cell-type specific expression of novel genes

With the advent of [single-cell sequencing](https://en.wikipedia.org/wiki/Single_cell_sequencing) it is now easy to examine transcriptome profiles at single cell level.
I am examining publicly available single cell RNA-Seq datasets to look for expression of novel genes in a cell specific manner.



### A deep-generative model to integrate massive bulk expression datasets



### MetaOmGraph: A tool for exploration of big omics data


### pyrpipe: Bioinformatics pipelines in pure-python


### orfipy: Fast and flexible ORF caller
