---
layout: page
title: Publications
date: 202-02-29T19:09:34+01:00
modified: 202-02-29T19:09:34+01:00
excerpt: "Arun Seetharam's publications"
tags: [Urminder Singh]
---

<span id='badgeCont419579' style='width:126px'>
<script src='http://labs.researcherid.com/mashlets?el=badgeCont419579&mashlet=badge&showTitle=false&className=a&rid=B-8899-2015'>
</script>
</span>
<section id="table-of-contents" class="toc">
  <header>
    <h3>Overview</h3>
  </header>
<div id="drawer" markdown="1">
*  Auto generated table of contents
{:toc}
</div>
</section><!-- /#table-of-contents -->

#### Please see my [Google scholar page](https://scholar.google.com/citations?user=yat-ghwAAAAJ&hl=en) for most up-to-date list of publications.

## Preprints

- **Urminder Singh**, Kyle Hernandez, Bruce Aronow, and Eve S Wurtele. African Americans and European Americans exhibit distinct gene expression patterns across tissues and tumors that are associated with immunologic and infectious functions and environmental exposures. _ResearchSquare_, 2020.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://assets.researchsquare.com/files/rs-88890/v1_stamped.pdf)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/urmi-21/COVID-DEA)

- **Urminder Singh**, Jing Li, Arun S Seetharam and Eve S Wurtele. pyrpipe: a python package for rna-seq workfows. _bioRxiv_, page 925818, 2020.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1101/2020.03.04.925818)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/urmi-21/pyrpipe)

- Arun S Seetharam, **Urminder Singh**, Jing Li, Priyanka Bhandary, Zebulun Arendsee, and Eve Syrkin Wurtele. Maximizing prediction of orphan genes in assembled genomes. _bioRxiv_, 2019.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1101/2019.12.17.880294)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/eswlab/orphan-prediction)

- Jing Li, **Urminder Singh**, Zebulun Arendsee, and Eve Syrkin Wurtele. Landscape of the Dark Transcriptome Revealed through Re-mining Massive RNA-Seq Data. _bioRxiv_, 2020.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://www.biorxiv.org/content/10.1101/671263v2.full)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/lijing28101/yeast_supplementary)

- Zebulun Arendsee, Andrew Wilkey, **Urminder Singh**, Jing Li, Manhoi Hur, and Eve Syrkin Wurtele. synder: inferring genomic orthologs from synteny maps. *BioRxiv*(2019): 554501.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://www.biorxiv.org/content/10.1101/554501v1)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/arendsee/synder)

## Peer-reviewed 

- **Urminder Singh**, and Eve S Wurtele. orfipy: a fast and flexible tool for extracting ORFs. _Bioinformatics_, 2020.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://www.biorxiv.org/content/10.1101/2020.10.20.348052v1)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/urmi-21/orfipy)

- **Urminder Singh**, Manhoi Hur, Karin Dorman, Eve Syrkin Wurtele. MetaOmGraph: a workbench for interactive exploratory data analysis of large expression datasets. *Nucleic acids research* 48.4 (2020): e23-e23.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://academic.oup.com/nar/article/48/4/e23/5709708)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/urmi-21/MetaOmGraph)

- **Urminder Singh** and Eve Syrkin Wurtele. Genetic novelty: How new genes are born. *Elife* 9 (2020): e55136.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://elifesciences.org/articles/55136)

- Zebulun Arendsee, Jing Li, **Urminder Singh**, Priyanka Bhandary, Arun Seetharam, and Eve Syrkin Wurtele. Fagin: synteny-based phylostratigraphy and finer classification of young genes. *BMC bioinformatics* 20.1 (2019): 1-14.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3023-y)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/arendsee/fagin)

- Ananya Neogy, Tushar Garg, Anil Kumar, Anuj K Dwivedi, Harshita Singh, **Urminder Singh**, Zeenu Singh, Kalika Prasad, Mukesh Jain, Shri Ram Yadav. Genome-Wide Transcript Profiling Reveals an Auxin-Responsive Transcription Factor, OsAP2/ERF-40, Promoting Rice Adventitious Root Development. *Plant and Cell Physiology* 60.10 (2019): 2343-2355.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://www.researchgate.net/profile/Tushar_Garg2/publication/334574454_Genome-Wide_Transcript_Profiling_Reveals_an_Auxin-Responsive_Transcription_Factor_OsAP2ERF-40_Promoting_Rice_Adventitious_Root_Development/links/5d5ec003299bf1b97cfedf51/Genome-Wide-Transcript-Profiling-Reveals-an-Auxin-Responsive-Transcription-Factor-OsAP2-ERF-40-Promoting-Rice-Adventitious-Root-Development.pdf)

- Zebulun Arendsee, Jing Li, **Urminder Singh**, Arun Seetharam, Karin Dorman, and Eve Syrkin Wurtele. phylostratr: A framework for phylostratigraphy. _Bioinformatics_, 35(19):3617--3627, 2019.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1093/bioinformatics/btz171)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/arendsee/phylostratr)

- **Urminder Singh**, Kushal Shah, Suman Dhar, Vinod Singh, and Annangarachari Krishnamachari. ORIS: An interactive software tool for prediction of replication origin in prokaryotic genomes. *Journal of Open Source Software* 4.40 (2019): 1589.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://joss.theoj.org/papers/10.21105/joss.01589.pdf)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/urmi-21/ORIS)

- **Urminder Singh**, Niraj Khemka, Mohan Singh Rajkumar, Rohini Garg, and Mukesh Jain. PLncPRO for prediction of long non-coding RNAs (lncRNAs) in plants and its application for discovery of abiotic stress-responsive lncRNAs in rice and chickpea. *Nucleic acids research* 45.22 (2017): e183-e183.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://academic.oup.com/nar/article/45/22/e183/4282667)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/urmi-21/PLncPRO)

- **Urminder Singh**, Sucheta Chauhan, A Krishnamachari, and Lovekesh Vig. Ensemble of deep long short term memory networks for labelling origin of replication sequences. *2015 IEEE International Conference on Data Science and Advanced Analytics (DSAA)*. IEEE, 2015.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://www.researchgate.net/profile/Sucheta_Chauhan3/publication/308837697_Ensemble_of_deep_long_short_term_memory_networks_for_labelling_origin_of_replication_sequences/links/5a7193b5aca272e425edde75/Ensemble-of-deep-long-short-term-memory-networks-for-labelling-origin-of-replication-sequences.pdf)


## Book chapters

- Niraj K Khemka, **Urminder Singh**, Anuj K Dwivedi, and Mukesh Jain. Machine Learning-Based Annotation of Long Noncoding RNAs Using PLncPRO. *Legume Genomics*. Humana, New York, NY, 2020. 253-260.
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/urmi-21/PLncPRO)














